lib/addons/apache-airflow/index.ts

```diff
@@ -217,6 +217,8 @@ function populateValues(clusterInfo: ClusterInfo, ns: KubernetesManifest, helmOp
             overwrite: true,
         });
 
+        efsResources.node.addDependency(ns);
```

Comment:
efs storage class is not a namespace resource - created at the global. So adding dependency to the namespace is irrelevant for it but it is revelant for the PVC. 

The error that we saw was: either chart or PVC or both are created before the storage class is created. 

We need the following dependencies:

efs storage class depends efs csi driver - it is a global resource so dependencies on ns, pvc, chart
PVC depends on storage class and ns
chart depends on PVC/storage class

Reply:
Am I interpreting this correctly:

For the chart, we would only need to add Dependency for the PVC - if I implement PVC node dependency on sc and ns, by transitivity, sc and ns would provision before the chart.

Reply:
Yeah, that seems right. The storage class may require dependency on the EFS CSI driver most likely. I am curious if CSI driver dependency is critical for this case, this will have to be tested.

Reply:
Is there a way to add CSI driver as dependency? addons are Construct promises, which do not use IDependable interface.

Reply:
I am a bit confused with this question since we had this conversation in the past, I attribute it to the case of Mondays. I believe [this](https://github.com/aws-quickstart/cdk-eks-blueprints/blob/c5ad95777e7db129916f03c18b9a07f8be0be3c4/lib/addons/jupyterhub/index.ts#L251) is your code based on the discussion we had.

Reply:
Case of the Mondays it is.